recipe/meta.yaml

```diff
@@ -27,9 +27,9 @@ requirements:
     - libgpg-error 1.42
     - libiconv 1.16  # [not linux]
     - libtool
```

Comment:
Should this be removed?

Reply:
Yes, good point. It's probably not needed as a host dependency, but I thought it is needed in the run section because of libltdl

Reply:
I just had a look at the `build.sh` and indeed maybe `libtool` is not necessary.

Reply:
See if it builds without?

Reply:
checking now

Reply:
turns out it is necessary during the build because it links against `libltdl` which is provided by `libtool`.